frontend/src/components/Footer.tsx

```diff
@@ -6,7 +6,6 @@ import {
   Address,
   Grid,
   Icon,
-  Logo,
```

Comment:
Not using the Truss `Logo` component, as it wraps the image in a grid that you cannot control.